Storage/ThirdParty/SwiftData.swift

```diff
@@ -798,7 +798,8 @@ class SDRow: Sequence {
 
         switch type {
         case SQLITE_NULL, SQLITE_INTEGER:
-            ret = Int(sqlite3_column_int64(statement.pointer, i))
+            //Everyone expects this to be an Int. On Ints larger than 2^31 this will lose information.
+            ret = Int(truncatingBitPattern: sqlite3_column_int64(statement.pointer, i))
```

Comment:
This should be okay. The largest numbers we store are the unix timestamps and those fit.